src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -16,7 +16,12 @@
 -->
 
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+<<<<<<< HEAD
+  <application>	
```

Comment:
I guess you don't know how to resolve merge conflicts. So this `<<<<<<< HEAD` means you pull some code and git gave you a message that there is a conflict between version of code which you have and version of code which is pulled. So whatever is between `<<<<<<< HEAD` and `=====` is what you had locally and whatever is b/w `====` and `>>>>>> b69962e2eea9d64bb28ed4a429459812e3004ddc ` is what came from git pull so you need to choose b/w one of them. Read more about git merge conflict resolution.